js/RotorPirates.html

```diff
@@ -0,0 +1,58 @@
+<html>
+    <head>
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy" crossorigin="anonymous">
+        
+        <link rel="stylesheet" href="style.css">
+        <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+
+        <script src="RotorPirates.js"></script>
+    </head>
+    <body>
+        <img src="http://monsterquads.eu/wp-content/uploads/2017/02/monsterquadsXR-log2.png" style="width:400px;margin-top:30px"/>
+        <h3>Rates Calculator</h3>
+        <div id="curve_chart" style="width: 900px; height: 500px; margin-top:-70px"></div>
+        <div class="rateBox">
+            <span class="block-title">RaceFlight Rates</span>
+            <div>
+                <span class="input-title">Rate:</span>
+                <span class="input-title">Expo:</span>
+                <span class="input-title">Acro+:</span>
+            </div>
+            <div>
+                <input type="number" name="rfRate" min="0" max="2000" value="400" class="input-field"/>
```

Comment:
As mentioned, Slider input (type="range") though I'm no web person so not sure how to finagle that in a pretty fashion.
Number inputs are fine but sliders too!!!!!

Reply:
I thought about it and to me this seems closer to UI in configurators for these FC's.

Reply:
It is (and feel free to keep that in there), but Sliders are where the real value is at :)  its difficult to play with the ranges of these values with increment buttons

Reply:
Will work on it

Reply:
Cool let me know :) Once you have it done i'll merge in and host on github pages :D